MoltenVK/MoltenVK/Commands/MVKCmdRenderPass.h

```diff
@@ -50,6 +50,7 @@ class MVKCmdBeginRenderPassBase : public MVKCommand {
 	MVKFramebuffer* _framebuffer;
 	VkRect2D _renderArea;
 	VkSubpassContents _contents;
+	MVKSmallVector<MVKImageView*, 8> _imagelessAttachments;
```

Comment:
This class probably ought to be templated, providing variants with 0, 1, and 8 attachments, to handle the common cases of not using imageless and using it with one attachment.

Reply:
> This class probably ought to be templated, providing variants with 0, 1, and 8 attachments, to handle the common cases of not using imageless and using it with one attachment.

Nice catch.